test/e2e/run_test.sh

```diff
@@ -208,6 +225,19 @@ while (( "$#" )); do
       IPV6=true
       shift
     ;;
+    --cleanup-kind)
+      CLEANUP_KIND=true
+      shift
+    ;;
+    --cleanup-registry)
+      CLEANUP_REGISTRY=true
+      shift
+    ;;
+    --cleanup-all)
```

Comment:
I think cleanup both kind and dockerregistry is meaningful, not only one of them